AzureCalling/app/src/main/java/com/azure/samples/communication/calling/external/calling/JoinCallConfig.java

```diff
@@ -3,27 +3,32 @@
 
 package com.azure.samples.communication.calling.external.calling;
 
+import com.azure.samples.communication.calling.helpers.JoinCallType;
+
 import java.io.Serializable;
 
 public class JoinCallConfig implements Serializable {
-    private final String groupId;
+    private final String joinId;
     private final boolean isMicrophoneMuted;
     private final boolean isCameraOn;
     private final String displayName;
+    private final JoinCallType callType;
 
     public JoinCallConfig(
-            final String groupId,
+            final String joinId,
             final boolean isMicrophoneMuted,
             final boolean isCameraOn,
-            final String displayName) {
-        this.groupId = groupId;
+            final String displayName,
+            final JoinCallType callType) {
+        this.joinId = joinId;
         this.isMicrophoneMuted = isMicrophoneMuted;
         this.isCameraOn = isCameraOn;
         this.displayName = displayName;
+        this.callType = callType;
     }
 
-    public String getGroupId() {
-        return groupId;
+    public String getJoinId() {
+        return joinId;
```

Comment:
We should consider applying these renaming changes in main branch first to minimize changes later on when merging.